bpf/lib/nat46.h

```diff
@@ -401,4 +410,38 @@ static inline int ipv6_to_ipv4(struct __sk_buff *skb, int nh_off,
 	return 0;
 }
 
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT64) int tail_ipv6_to_ipv4(struct __sk_buff *skb)
+{
+	int ret = ipv6_to_ipv4(skb, 14, htonl(LXC_IPV4));
+	if (IS_ERR(ret))
+		return ret;
+
+	cilium_trace_capture(skb, DBG_CAPTURE_AFTER_V64, skb->ingress_ifindex);
+
+	skb->cb[CB_NAT46_STATE] = NAT64;
+
+	tail_call(skb, &cilium_calls, CILIUM_CALL_IPV4);
+	return DROP_MISSED_TAIL_CALL;
+}
+
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT46) int tail_ipv4_to_ipv6(struct __sk_buff *skb)
```

Comment:
Same here. You rely on `LXC_ID` which is only valid in `bpf_lxc.c` context. Let's move this tail call wrapper into `bpf_lxc.c`